src/Kmd.Logic.CitizenDocuments.Client/CitizenDocumentsOptions.cs

```diff
@@ -15,10 +13,8 @@ public sealed class CitizenDocumentsOptions
         /// <remarks>
         /// This option should not be overridden except for testing purposes.
         /// </remarks>
-        public Uri Serviceuri { get; set; } = new Uri("https://gateway.kmdlogic.io/cpr/v1");
-
-        public string SubscriptionId { get; set; } = "";
-
+        public Uri Serviceuri { get; set; } = new Uri(string.Empty);
```

Comment:
Why is this a better default?

Reply:
As these values will be set later by developers. So here we are just putting placeholder